tests/functional/monitoring/conftest.py

```diff
@@ -1177,7 +1177,7 @@ def change_client_version():
         nonlocal client
         nonlocal original_cluster
         # run_time of operation
-        run_time = 60 * 3
+        run_time = 60 * 7
```

Comment:
@fbalak why is this value set at 60*7 ? Could you please clarify to me?

Reply:
StorageClientHeartbeatMissed - description has 120(s) for Warning and 300(s) for Critical.

Expressions in Prometheus rules confirming. Waring expression for reference.
`[(time() - 120) > (ocs_storage_client_last_heartbeat > 0)](https://console-openshift-console.apps.ibm-baremetal1.qe.rh-ocs.com/monitoring/query-browser?query0=(time()%20-%20120)%20%3E%20(ocs_storage_client_last_heartbeat%20%3E%200))`

![image](https://github.com/red-hat-storage/ocs-ci/assets/61454420/0d3a1844-48cf-4da5-9289-3ffbc8481b8c)
![image](https://github.com/red-hat-storage/ocs-ci/assets/61454420/53b58ec5-e170-46c5-bd34-76a9859c1002)

StorageClientIncompatibleOperatorVersion immediate as far as I understand. No interval in the description. 

So all together should be 420sec enough, which is the same as Filip set here